### ISSUE TITLE
Bug: IR Device Commands not being sent

### ISSUE THREAD

**Issue by deflao** [OpenWonderLabs/homebridge-switchbot#551]
### Describe The Bug

Light IR and DIY Light IR don't send signal. I have 2 IR device and it seems to work fine until I updated to the latest version (v2.2.1)
The Fan IR seems to works fine but I'm having trouble with Light IR. If I disable allow on/off change to be pushed, then it doesn't work at all. And when I enable allow off change to be pushed then I can turn on the device but can't turn it off.

### To Reproduce

Turn on/off the device from Home app

### Expected behavior

Turn off the device

### Relevant log output

```shell
[20/10/2022, 22:55:29] [SwitchBot] [DEBUG] DIY Light: Light updateCharacteristic On: false
[20/10/2022, 22:55:30] [SwitchBot] [DEBUG] DIY Light: Light statusCode: 200
[20/10/2022, 22:55:30] [SwitchBot] [DEBUG] DIY Light: Light Unknown statusCode.
[20/10/2022, 22:55:30] [SwitchBot] [DEBUG] DIY Light: Light d: {"statusCode":100,"body":{"items":[{"deviceID":"02-202112122245-99096069","code":100,"status":{"power":1},"message":"success"}]},"message":"success"}
[20/10/2022, 22:56:44] [RokuTV] Getting Power State: Ready
[20/10/2022, 22:56:46] [SwitchBot] [DEBUG] DIY Light: Light On: false
[20/10/2022, 22:56:46] [SwitchBot] [DEBUG] DIY Light: Light pushLightOffChanges On: true, allowPushOff: undefined
```

### Config for homebridge-switchbot

```shell
"name": "SwitchBot",
            "credentials": {
                "token": "xxxx",
                "secret": "xxxx",
                "notice": "Keep your token a secret!"
            },
            "options": {
                "devices": [
                    {
                        "deviceId": "xxxx",
                        "configDeviceType": "Curtain",
                        "connectionType": "OpenAPI",
                        "curtain": {
                            "hide_lightsensor": true
                        }
                    },
                    {
                        "deviceId": "xxxx",
                        "configDeviceType": "Curtain",
                        "connectionType": "OpenAPI",
                        "curtain": {
                            "hide_lightsensor": true
                        }
                    },
                    {
                        "deviceId": "xxxx",
                        "configDeviceType": "Humidifier",
                        "connectionType": "OpenAPI",
                        "humidifier": {
                            "hide_temperature": true
                        }
                    },
                    {
                        "deviceId": "xxxx",
                        "configDeviceType": "Humidifier",
                        "connectionType": "OpenAPI",
                        "humidifier": {
                            "hide_temperature": true
                        }
                    }
                ],
                "irdevices": [
                    {
                        "deviceId": "xxxx",
                        "configDeviceName": "Fan",
                        "configRemoteType": "Fan",
                        "connectionType": "OpenAPI",
                        "logging": "standard"
                    },
                    {
                        "deviceId": "xxxx",
                        "configDeviceName": "Light",
                        "configRemoteType": "Light",
                        "connectionType": "OpenAPI",
                        "allowPushOn": false,
                        "allowPushOff": true,
                        "external": true,
                        "logging": "debug"
                    }
                ]
            },
            "platform": "SwitchBot"
```

### Device & Model

Raspberry Pi 4 Model B

### Node.js Version

v14.20.0

### NPM Version

v8.19.1

### Homebridge Version

v1.5.0

### Homebridge Switchbot Plugin Version

v2.2.1

### Homebridge Config UI X Plugin Version

v4.50.1

### Operating System

Raspbian GNU/Linux Buster (10)

**Comment by robberburn** [comment]
Same here, last working 1.15.0

**Comment by donavanbecker** [comment]
Fixed in v2.2.2
